### ISSUE TITLE
Docker cant run image openwrtorg/rootfs:x86-64

### ISSUE THREAD

**Issue by ZoZhang** [openwrt/docker#48]
Hello, I encountered the following problems when using Docker. Can you help me?

#### Error by docker build
````
Collected errors:
 * satisfy_dependencies_for: Cannot satisfy the following dependencies for luci-mod-admin-full:
 * 	kernel (= 5.4.38-1-9dea1f6ed59a3231b5bc094092a9414a)
 * opkg_install_cmd: Cannot install package luci-mod-admin-full.
````

#### Dockerfile 
```
FROM openwrtorg/rootfs:x86-64

MAINTAINER Zhao ZHANG <zo.zhang@gmail.com>

USER root

RUN mkdir /var/lock && \
    opkg update && \
    opkg install uhttpd luci-base luci-mod-admin-full luci-theme-bootstrap luci-i18n-base-en luci-i18n-firewall-en luci-i18n-base-fr luci-i18n-firewall-fr && \
    uci set luci.ccache.enable=0 && \
    uci commit luci

EXPOSE 80
```

**Comment by aparcar** [comment]
Can you please run `uname -a` in the docker container? I'm wondering if it is because the host system uses a different kernel than opkg excepts, but that's may be unrelated. I'll keep investigating

**Comment by jonglezb** [comment]
Could it be that the rootfs snapshot of the docker image is too old for the current luci snapshot from downloads.openwrt.org?

I'm actually surprised that luci depends on the kernel package, but the error looks like the common version mismatch that happens when using older snapshots and installing newer packages.

A workaround is to use `opkg --force-depends` to ignore the dependency, but I guess the proper fix would be to rebuild the rootfs docker image regularly.

**Comment by ynezz** [comment]
> but I guess the proper fix would be to rebuild the rootfs docker image regularly.

They're build every day at 4am via GitLab [pipeline](https://gitlab.com/openwrt/docker/pipelines/143152842) [schedules](https://gitlab.com/openwrt/docker/pipeline_schedules). Currently it's broken as one of the tests for image builder is currently [failing]](https://gitlab.com/openwrt/docker/-/jobs/540067478) due to the x86/64 image generation refactoring ` ls: cannot access './bin/targets/x86/64/*combined-squashfs.img.gz': No such file or directory` and we're simply not aware about it.

It's quite strange, that GitLab doesn't send out failure emails in case of scheduled pipeline failures like it does for ordinary after the `git push` pipeline runs.

**Comment by ynezz** [comment]
> It's quite strange, that GitLab doesn't send out failure emails in case of scheduled pipeline failures like it does for ordinary after the git push pipeline runs.

Seems like this needs to be explicitly configured in https://gitlab.com/openwrt/docker/-/services/pipelines_email/edit

**Comment by aparcar** [comment]
GitHub automatically closed this issue by merging PR #50, please reopen if the issue is still existing...

**Comment by ynezz** [comment]
I've just tested it and the `opkg install uhttpd luci-base luci-mod-admin-full luci-theme-bootstrap luci-i18n-base-en luci-i18n-firewall-en luci-i18n-base-fr luci-i18n-firewall-fr` worked.
